lib/Elastica/Index.php

```diff
@@ -295,7 +293,7 @@ public function exists()
      */
     public function createSearch($query = '', $options = null)
     {
-        $search = new Search($this->getClient());
+        $search = new Search($this->getClient(), $this->getClient()->getResultSetBuilder());
```

Comment:
What do you exactly mean by "set a creation time"? You mean inside the search object (which seems to be the case also with an optional param) or here inside createSearch? 

As createSearch is "only"  a proxy method it should probably provide all options that can be passed to Search itself. So I somehow like the idea of providing an additional third param which is optional. 

I can see your point about simplifying the code on our side and preventing bugs, but it makes it harder for beginners to start with Elastica. Some "magic" is ok from my point of view, as long as it can be overloaded by setting the specific params.